src/renderers/dom/client/wrappers/ReactDOMTextarea.js

```diff
@@ -66,12 +65,14 @@ var ReactDOMTextarea = {
       '`dangerouslySetInnerHTML` does not make sense on <textarea>.'
     );
 
-    // Always set children to the same thing. In IE9, the selection range will
-    // get reset if `textContent` is mutated.
```

Comment:
Looked into it.  Appears that IE9 does indeed get upset if you set textContent to the same value it already had.  But this was totally the wrong place for a comment to be talking about that; applied a hack in `setTextContent.js` which is a much more appropriate place to deal with this.